src/Files.App/Data/Items/WidgetDriveCardItem.cs

```diff
@@ -32,6 +32,12 @@ public async Task LoadCardThumbnailAsync()
 				true,
 				IconOptions.ReturnIconOnly | IconOptions.UseCurrentScale);
 
+			if (result is null)
+			{
+				using var thumbnail = await DriveHelpers.GetThumbnailAsync(Item.Root);
+				result ??= await thumbnail.ToByteArrayAsync();
```

Comment:
There's already null-check.

```suggestion
				result = await thumbnail.ToByteArrayAsync();
```

Reply:
We're trying to move away from GetThumbnailAsync. Let me know if you need assistance but I think we should investigate which flags are being in `DriveHelpers.GetThumbnailAsync` so that we use similar flags with the Win32 api.

Reply:
In the meantime we can merge this to fix the issue.